src/main/java/com/github/johrstrom/util/ServerInstantiator.java

```diff
@@ -0,0 +1,56 @@
+package com.github.johrstrom.util;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.jetty.server.Server;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ServerInstantiator {
+	private static final Logger log = LoggerFactory.getLogger(ServerInstantiator.class);
+	private static Map<Integer, Server> server_instances;
+
+	/**
+	 * Retrieves or creates the server
+	 * 
+	 * @param port
+	 */
+	public static Server getInstance(int port) {

```

Comment:
I wish we hadn't pulled this in so fast.  I think this was a reaction of the fact that instantion only occured on the client erroneously.  The PrometheusListener already implements [NoThreadClone](https://jmeter.apache.org/api/org/apache/jmeter/engine/util/NoThreadClone.html) which makes it a singleton. 

I don't see any use case ever to have more than one Server booted.